cookbooks/crystal_preferred_orientation_olivine_fraters_billen_2021/cpo_pole_figure_config.toml

```diff
@@ -0,0 +1,15 @@
+base_dir = "path/to/where/you/run/aspect/input"
```

Comment:
Why do you include this file twice, once here and once inside the `doc` directory? Delete one of them.